base/logging_context.go

```diff
@@ -206,6 +209,33 @@ func KeyspaceLogCtx(parent context.Context, bucketName, scopeName, collectionNam
 	return LogContextWith(parent, &newCtx)
 }
 
+type userIDDomain string
+
+const (
+	UserDomainSyncGateway userIDDomain = "sgw"
+	UserDomainCBServer    userIDDomain = "cbs"
+	UserDomainBuiltin                  = "builtin" // internal users (e.g. SG bootstrap user)
```

Comment:
I don't think this will matter, but:

```suggestion
	UserDomainBuiltin   userIDDomain  = "builtin" // internal users (e.g. SG bootstrap user)
```